tests/OrganizationsMetadataTest.php

```diff
@@ -119,6 +119,8 @@ public function testSuperAdminCanManageMetadata(string $provider): void
         $this->assertCount(2, $metadataArray);
         $this->validateMetadataEquality(self::TEST_METADATA[1], $metadataArray[0], $provider);
         $this->validateMetadataEquality(self::TEST_METADATA[0], $metadataArray[1], $provider);
+
+        $this->deleteMetadata($this->client, $metadataArray);
```

Comment:
Tu by som teda pridal, test ze ked mu odoberies toho super admina tak $this->cannotManageUserMetadata a cannotManageSystemMetadata